### PR TITLE
ci: Fixes multi arch build

### PR DESCRIPTION
**Pull request by soluchok**
Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

File: images/vct/Dockerfile [hunk 1]
```diff
@@ -31,9 +31,7 @@ RUN echo ${TARGETARCH}
 
 RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o build/bin/vct cmd/vct/main.go
 
-ARG ALPINE_VER
-
-FROM alpine:${ALPINE_VER} as base
+FROM alpine as base
```
Comment (troyronda) [inline images/vct/Dockerfile:34]: i would actually do scratch as base.

copy needed files (eg ca-certificates) from alpine.

File: images/vct/Dockerfile [hunk 2]
```diff
@@ -31,10 +31,10 @@ RUN echo ${TARGETARCH}
 
 RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o build/bin/vct cmd/vct/main.go
 
-ARG ALPINE_VER
+FROM scratch
 
-FROM alpine:${ALPINE_VER} as base
 LABEL org.opencontainers.image.source https://github.com/trustbloc/vct
-RUN apk add -U --no-cache ca-certificates
+
```
Comment (troyronda) [inline images/vct/Dockerfile:37]: And the next step is to copy needed files from an alpine multi-stage builder (e.g., ca-certificates).

**Review by troyronda — APPROVED** [review]

File: images/vct/Dockerfile [hunk 3]
```diff
@@ -29,12 +22,13 @@ ARG TARGETARCH
 RUN echo ${TARGETOS}
 RUN echo ${TARGETARCH}
 
-RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o build/bin/vct cmd/vct/main.go
+RUN CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o /usr/bin/vct cmd/vct/main.go
 
-ARG ALPINE_VER
+FROM scratch
+
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```
Comment (troyronda) [inline images/vct/Dockerfile:29]: I would personally use a second multi-stage pure alpine image to copy these files from.

Comment (soluchok) [reply images/vct/Dockerfile:29]: ok, I will do that in the follow up PR

**Pull request merged**